v2/cli/event_commands.go

```diff
@@ -63,6 +63,12 @@ var eventCommand = &cli.Command{
 					Usage: "If set, will additionally abort and cancel events with " +
 						"their worker in a RUNNING phase",
 				},
+				&cli.BoolFlag{
+					Name:    flagStarting,
+					Aliases: []string{"r"},
```

Comment:
Is this alias intended?  Noticing it isn't added to the other instances of this new flag.

Reply:
That's a copy/paste error. Good eye. Fixing.

Reply:
Fixed in https://github.com/brigadecore/brigade/pull/1186/commits/65c74ab7cf9ac34afbf67d5871882751aefb9cae